### DISCUSSION TITLE
Yearly budget...

### DISCUSSION THREAD

**Discussion by ashfaqfarooqui** [ananthakumaran/paisa#269] [Ideas]
HI,
Thanks for this app, its solves many of the pain points I had.

One thing I would prefer is to have the budget page only calculate for the current year. My use case is as follows, At the start of the year I create a rolling budget for the whole year. This budget is rolling and every month will have rollover of the accounts. However, I wouldn't want them to roll over to the next year. At the end of the year I can get an idea how well the budget fit with the actual spending.

Would it be possible to limit the budget reports only for that year.

**Comment by parthvzala** [comment]
Any solution to this?

**Reply by kvineet** [reply]
at the end of the year, manually move the remaining balance out to savings account?

Basically you are utilizing the remaining budget to save (or party)
